src/css/_docs.scss

```diff
@@ -1,29 +1,35 @@
+$mobile-large:  640px;
```

Comment:
So I was looking at a weird half breakpoint, but Ben said something more stable might be coming in Phablet form. Wasn't a huge issue. Will remove for now.

I'll remove the fixed height.